### ISSUE TITLE
'Tag' widget's vocabulary definition not working

### ISSUE THREAD

**Issue by jenil777007** [annotorious/annotorious#62]
I want the user to only select one option from the defined vocabulary. And as per the guidelines available in the official documentation, I have initialised my object as below,

```
var anno = Annotorious.init({
          image: "hallstatt",
          widgets: [
            { widget: "TAG", vocabulary: ["Animal", "Building", "Waterbody"] }
          ]
});
```

But the result is allowing me to type anything in the TAG widget.

Reproduction link:
[https://codesandbox.io/s/annotoriousvanilla-954fu](url)

**Comment by rsimon** [comment]
I'm afraid that's the way the tag widget is intended to work. The vocabulary is only there to provide suggestions/autocomplete functionality. But, by design, the user is still able to type anything they want. Restricting the options to __only__ the suggested terms is currently not supported.

**Comment by jenil777007** [comment]
Thanks for the quick response @rsimon . But if you see the reproduction link even the suggestions/autocomplete is not working. Am I missing something?

**Comment by rsimon** [comment]
Ah - sorry. Yes, this was broken in the 2.1.2 release. But is now fixed in the most recent release ([2.1.3](https://github.com/recogito/annotorious/releases/tag/v2.1.3), released just two days ago). Looks like this works properly in your example as soon as you upgrade to the 2.1.3 script.

**Comment by jenil777007** [comment]
Thanks again @rsimon . 

**_Upgrading the script to 2.1.3 resolved the issue_**. 

Closing the issue.
